src/playground/sample_data_ingestion/sample_data_ingestor/tests/test-files/valid-template-inputs.json

```diff
@@ -0,0 +1,42 @@
+{
+    "jinput": {
+        "year": "year",
+        "random number": "integer",
+        "email": "email"
+    },
+
+    "jinput2": {
+        "properties": {
+            "first name":    { "type" : ["first_name"]},
+            "last name":     { "type" : "last_name"},
+            "companies":{ "type" : ["array", "company", 5] }
+        }
+    },
+
+    "jinput3": {
```

Comment:
can we provide a good name, not 1,2,3

Reply:
Done, refactored test case variable names